### ISSUE TITLE
csi driver: invalid syntax for size

### ISSUE THREAD

**Issue by rayjanoka** [vitalif/vitastor#50]
Hey, I think I have everything setup on the latest version, but I'm running into an error with the csi driver about the `size`. I'm also sending `poolId: "2"` but I see that value isn't being added to the CLI command. 

```csi-vitastor-provisioner-6494b48b8-b4m4z csi-vitastor E0317 02:22:30.396662       1 
controllerserver.go:136] vitastor-cli create pvc-00fca7d6-cdf6-48f2-9065-f5cc38a10eab 
-s � --pool  --etcd_address 10.10.0.161:2379,10.10.0.162:2379,10.10.0.163:2379,
10.10.0.164:2379,10.10.0.165:2379,10.10.0.166:2379 --etcd_prefix /vitastor 
failed: Invalid syntax for size: �
```

```csi-vitastor-provisioner-6494b48b8-b4m4z csi-vitastor I0317 02:17:29.363359       1 
controllerserver.go:145] received controller create volume request {
  "capacity_range": {
    "required_bytes": 10737418240
  },
  "name": "pvc-00fca7d6-cdf6-48f2-9065-f5cc38a10eab",
  "parameters": {
    "csi.storage.k8s.io/pv/name": "pvc-00fca7d6-cdf6-48f2-9065-f5cc38a10eab",
    "csi.storage.k8s.io/pvc/name": "test-vitastor-pvc",
    "csi.storage.k8s.io/pvc/namespace": "vitastor-system",
    "etcdPrefix": "/vitastor",
    "etcdUrl": "10.10.0.161:2379,10.10.0.162:2379,10.10.0.163:2379,10.10.0.164:2379,10.10.0.165:2379,10.10.0.166:2379",
    "etcdVolumePrefix": "",
    "poolId": "2"
  },
  "volume_capabilities": [
    {
      "AccessType": {
        "Mount": {
          "fs_type": "ext4"
        }
      },
      "access_mode": {
        "mode": 1
      }
    }
  ]
}
```

**Comment by rayjanoka** [comment]
I downgraded the csi driver to v0.8.5 and my pvc came online!

```
➜ kg pvc
NAME                      STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
test-vitastor-pvc         Bound    pvc-00fca7d6-cdf6-48f2-9065-f5cc38a10eab   10Gi       RWO            vitastor       5h28m
test-vitastor-pvc-block   Bound    pvc-37282c52-d02c-4a09-9a80-ae3546cf053a   10Gi       RWX            vitastor       29m
```

**Comment by vitalif** [comment]
Oops. Sorry :)

**Comment by vitalif** [comment]
I pushed a fix and rebuilt 0.8.6, can you retest it ? :)
